Update debug logging documentation (#40421)
home-assistant/home-assistant.github.io: source/_docs/configuration/troubleshooting.markdown

To enable debug logging for a specific integration, follow these steps:

1. Go to {% my integrations title="**Settings** > **Devices & services**" %}.
2. Select the integration for which you want to enable debug logging.
3. In the top right of the page, open the three dots {% icon "mdi:dots-vertical" %} menu, and select **Enable debug logging**.

    <p class='img'>
      <img src='/images/docs/configuration/enable-debug-logging.png' alt='Screenshot showing the Enable debug logging button on an integration detail page'>
      Screenshot showing the <b>Enable debug logging</b> menu item.
    </p>
4. To see the error in the logs, you need to reproduce the error. Continue with the steps on [disabling debug logging and download logs](#disable-debug-logging-and-download-logs).

### Disable debug logging and download logs

> Once you enable debug logging, you ideally need to make the error happen. Run your automation, change up your device or whatever was giving you an error and then come back and disable the debug logging. Disabling the debug logging is the same as enabling, but now the menu option says **Disable Debug Logging**. After you disable it, you will be automatically prompted you to download your log file. Save this to a safe location to upload later.

Reviewer: ```suggestion
Once you enable debug logging, you ideally need to make the error happen. Run your automation, change up your device or whatever was giving you an error and then come back and disable the debug logging. Disabling the debug logging is the same as enabling, but now the menu option says **Disable debug logging**. After you disable it, you will be automatically prompted you to download your log file. Save this to a safe location to upload later.
```

1. Go to {% my integrations title="**Settings** > **Devices & services**" %}.

> 2. Select the integration card to open the detail page of the integration for which you want to enable debug logging.

Reviewer: ```suggestion
2. Select the integration for which you want to enable debug logging.
```

> 3. In the top right of the page, open the options menu, and select **Enable Debug Logging**.

Reviewer: ```suggestion
3. In the top right of the page, open the three dots {% icon "mdi:dots-vertical" %} menu, and select **Enable debug logging**.
```
